lte/gateway/python/magma/pipelined/app/enforcement_stats.py

```diff
@@ -577,6 +577,8 @@ def get_stats(self, cookie: int = 0, cookie_mask: int = 0):
         Use Ryu API to send a stats request containing cookie and cookie mask, retrieve a response and 
         convert to a Rule Record Table
         """
+        if not self._datapath:
```

Comment:
Should this be a separate PR?

Reply:
It's a minor adjustment to handle an edge case, I can make it a separate PR if desired but I thought it would be easier to just make the fix and put it in with this PR.

Reply:
Let's separate it out :) 
Each PR should handle independent changes
Also, I think if we separate it out, we can get it into master faster as it only requires @koolzz 's approval.